wmf/handlers.go

```diff
@@ -2047,9 +2053,25 @@ func (r *Handler) Language(resp http.ResponseWriter, req *http.Request) {
 	return
 }
 
+func (self *Handler) SecureHeaders(resp http.ResponseWriter) {
+	if resp != nil {
+		resp.Header().Set("Strict-Transport-Security", "max-age=15552000")
+		resp.Header().Set("X-Frame-Options", "deny")
+		resp.Header().Set("Frame-Options", "deny")
+		resp.Header().Set("X-XSS-Protection", "1; mode=block")
+		resp.Header().Set("X-Content-Type-Options", "nosniff")
+		resp.Header().Set("X-Content-Security-Policy", "Content-Security-Policy: default-src 'self'")
```

Comment:
Nope, misread the example form from https://www.owasp.org/index.php/List_of_useful_HTTP_headers